ntoskrnl/mm/ARM3/pool.c

```diff
@@ -680,8 +680,7 @@ MiAllocatePoolPages(IN POOL_TYPE PoolType,
     //
     // Allocations of less than 4 pages go into their individual buckets
     //
-    i = SizeInPages - 1;
-    if (i >= MI_MAX_FREE_PAGE_LISTS) i = MI_MAX_FREE_PAGE_LISTS - 1;
+    i = min(SizeInPages, MI_MAX_FREE_PAGE_LISTS) - 1;
```

Comment:
Why do you change this ? It's not a functional change, is it ?

Reply:
While doing changes, I spent more time understanding what's going on here than it should be (according to me, at least). And decided to change it to a more clear code :)